apps/studio/package.json

```diff
@@ -112,6 +112,7 @@
     "@chakra-ui/cli": "^2.4.1",
     "@isomer/eslint-config": "*",
     "@isomer/prettier-config": "*",
+    "@isomer/storybook-config": "*",
```

Comment:
we have moved it into package exported config for possible use across other packages in the monorepo e.g. components?

Reply:
yup. especially if we want to reuse the same viewports and chromatic modes. Should exploit being in a monorepo